--- clara-20031214/Makefile.orig	2004-04-02 15:26:51.730247326 +0200
+++ clara-20031214/Makefile	2004-04-02 15:27:14.200596651 +0200
@@ -88,7 +88,7 @@
 #
 # Add or remove flags if necessary:
 #
-CFLAGS = $(INCLUDE) -g -Wall $(COPTS)
+CFLAGS = $(INCLUDE) $(RPM_OPT_FLAGS) $(COPTS)
 #CFLAGS = $(INCLUDE) -g -Wall -pedantic $(COPTS)
 #CFLAGS = $(INCLUDE) -g -O2 -Wall $(COPTS)
 #CFLAGS = $(INCLUDE) -g -O2 -pedantic $(COPTS)
